web/src/components/groupView/groupView.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import './groupView.css'
+
+class GroupView extends Component {
+  render() {
+    return (
+      <>
+        <h2>{ this.props.groupInfo.name }</h2>
+        <h4>Tutor:</h4>
+        { this.props.groupInfo.tutor.name + " " + this.props.groupInfo.tutor.surname}
```

Comment:
We could also show the members of the group, since that info is returned by the group endpoint

Reply:
It wasn't a requirement in the first place and since it's a momentary recording of the group that can't be updated I'm not sure how meaningful that would be.